README.md

```diff
@@ -177,25 +181,33 @@ Other potentially useful environment variables may be found in `setup.py`.
 
 #### Install Dependencies
 
-Common
+**Common**
+
 ```bash
-conda install astunparse numpy ninja pyyaml mkl mkl-include setuptools cmake cffi typing_extensions future six requests dataclasses
```

Comment:
This sounds too complicated. Can we keep the common installation path, and then add a footnote about AppleM1/aarch64 machines?

Reply:
Which part feels complicated? Moving the mlk installation bits to the OS specific sections?

This may make copy/pasting a little more tedious for someone who's regularly using these instructions to install pytorch (they have to copy/paste one extra line now), but for someone who's installing pytorch from source for the first time I would argue that these instructions are simpler.  They just keep going from top to bottom and copy/pasting the commands that apply to them.

Reply:
I like that idea, and it sounds similarly shaped to the ask in https://github.com/pytorch/pytorch/issues/72556

Reply:
Attempting to give it a fix here! https://github.com/pytorch/pytorch/pull/91861